### PR TITLE
Feature/add points

### PR DESCRIPTION
**Pull request by chudaol**

### CODE REVIEW

File: enums/markerStatus.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default {
+  TODO: 'TODO',
```
Comment (STsaruk) [inline enums/markerStatus.js:2]: what does this markerStatus enum do? 
Maybe it makes sense to make an object with:  
`export default {
  TODO: {
     name: 'TODO',
     color: 'red'
  },
}`

then we can get rid of computed color and use it elsewhere. What do you think?

File: components/MapMarker.vue [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      }
+    },
+    data () {
+      return {
+        markerOptions: {
+          stroke: true,
+          opacity: 0.6,
+          weight: 100,
+        },
+        markerStatus
```
Comment (STsaruk) [inline components/MapMarker.vue:75]: you import markerStatus and also created a variable in the data with the same name - is this done intentionally?

**Review by Datikzz — APPROVED** [review]

File: components/MapMarker.vue (hunk 2 above)
Comment (Datikzz) [reply components/MapMarker.vue:75]: yes, its the way to use imported constant markerStatus in template, 

diff button is rendered on specific marker status

Comment (STsaruk) [reply components/MapMarker.vue:75]: got it, resolved

**Review by STsaruk — APPROVED** [review]
LGTM

File: enums/markerStatus.js (hunk 1 above)
Comment (chudaol) [reply enums/markerStatus.js:2]: i left the enum but also added the status color map, so we don't need to compute the color. thanks!

**Pull request merged**